### PR TITLE
Added project and about sections

### PR DESCRIPTION
**Pull request by Ghostman747**
ADD PROJECT AND ABOUT ME SECTION
-Added the list of projects and pictures of the projects and included deadlinks to the projects
-Ordered the project cards using grids and flexboxes
-Styled the various card items
-Added information about me
-Added a deadlink to my resume
-styled the about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Project list and about section</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div id="canvas">
```
Comment (rotimiazeez) [inline index.html:14]: - [ ] Kindly make sure the container of the section is a `section` tag 👍🏽
- [ ] Please note that the main container should have the `display: grid`property.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @Ghostman747 👋, 
## Good job so far!
### Summary 👏
- No linter errors ✔
- Good commit message 👍
- Descriptive PR message😊
- Good Readme 🚀
### There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please make sure your project is the continuation of the first milestone and not a different project entirely as seen on the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)  page
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (rotimiazeez) [reply index.html]: I understood your point, you can remove the `div` tags and start with a section tag with the class `project-list` as you already have 👍. Please don't forget make this the continuation of your `add toolbar and introduction` milestone. 💪🏽

Comment (Ghostman747) [reply index.html]: Already done thank you.

**Review by keneogu — APPROVED** [review]
## Status: Approved ✔️
Hi @Ghostman747,

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations! 🎉💯🌟


- The project is set up correctly ✔️
- Descriptive Pull Request title and summary✔️
- No linter errors ✔️

Cheers...Happy coding!..and keep soaring higher! 💻 🍷 🚀

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

-------

**Comment by Ghostman747** [comment]
Thank you.

**Pull request merged**